acceptancetests/assess_caas_deploy_charms.py

```diff
@@ -63,55 +63,40 @@ def assess_caas_charm_deployment(caas_client):
     if not caas_client.check_cluster_healthy(timeout=60):
         raise JujuAssertionError('k8s cluster is not healthy because kubectl is not accessible')
 
-    # add caas model for deploying caas charms on top of it
-    model_name = 'testcaas'
+    model_name = caas_client.client.get_controller_uuid() + '-testcaas'
     k8s_model = caas_client.add_model(model_name)
 
-    sc_name = caas_client.default_storage_class_name
-    # ensure storage pools for caas operator using default sc.
-    k8s_model.juju(
-        'create-storage-pool',
-        ('operator-storage', 'kubernetes', 'storage-class=%s' % sc_name)
-    )
-
-    # ensure storage pools for mariadb
-    # TODO(ycliuhw): remove storage-pool setup, because Juju takes care of it now.
-    # And add tests for deploy with & without storage setup.
-    mariadb_storage_pool_name = 'mariadb-pv'
-    k8s_model.juju(
-        'create-storage-pool',
-        (mariadb_storage_pool_name, 'kubernetes', 'storage-class=%s' % sc_name)
-    )
-
     k8s_model.deploy(
-        charm="cs:~juju/gitlab-k8s-0",
+        charm="cs:~juju/mediawiki-k8s-3",
         config='juju-external-hostname={}'.format(external_hostname),
     )
 
     k8s_model.deploy(
         charm="cs:~juju/mariadb-k8s-0",
-        storage='database=100M,{}'.format(mariadb_storage_pool_name),
     )
 
-    k8s_model.juju('relate', ('mariadb-k8s', 'gitlab-k8s'))
-    k8s_model.juju('expose', ('gitlab-k8s',))
-    k8s_model.wait_for_workloads(timeout=3600)
+    k8s_model.juju('relate', ('mediawiki-k8s:db', 'mariadb-k8s:server'))
+    k8s_model.juju('expose', ('mediawiki-k8s',))
+    k8s_model.wait_for_workloads(timeout=600)
 
     def success_hook():
         log.info(caas_client.kubectl('get', 'all', '--all-namespaces'))
 
     def fail_hook():
         success_hook()
         log.info(caas_client.kubectl('get', 'pv,pvc', '-n', model_name))
+        caas_client.ensure_cleanup()
 
-    url = '{}://{}/{}'.format('http', external_hostname, 'gitlab-k8s')
+    url = '{}://{}'.format(
+        'http', external_hostname,
+        # 'mediawiki-k8s',  # TODO(ycliuhw): enable relative path once problem solved.
```

Comment:
Is there a bug we can refer to here? 
Should say what the problem is.

Reply:
this problem is /mediawiki-k8s returns 404, seems we need overwrite target in ing config.
But I am not very sure yet.